squad/api/rest.py

```diff
@@ -1096,6 +1096,35 @@ def definition(self, request, pk=None):
         definition = self.get_object().definition
         return HttpResponse(definition, content_type='text/plain')
 
+    @action(detail=True, methods=['post'], suffix='resubmit')
+    def resubmit(self, request, **kwargs):
+        testjob = self.get_object()
+        if testjob.resubmit():
+            # find latest child of this job
+            testjob.resubmitted_jobs.last()
+            data = {"message": "OK", "url": rest_reverse('testjob-detail', args=[testjob.pk], request=request)}
+            return Response(data, status=status.HTTP_200_OK)
+        return Response({'message': "NOK"}, status=status.HTTP_200_OK)
```

Comment:
What's "NOK"? Not OK?

Reply:
Yes, it's "Not OK". Should we use sth else? "Error"?

Reply:
I guess "Error" is more intuitive. Maybe a small description as well?